packages/quick_actions/quick_actions_ios/example/ios/RunnerUITests/RunnerUITests.swift

```diff
@@ -5,6 +5,9 @@
 import XCTest
 
 private let elementWaitingTime: TimeInterval = 30
+private let quickActionPressDuration: TimeInterval = 1.5
+// Max number of tries to open the quick action menu if failed.
```

Comment:
maybe add a short explanation why we need to retry?

Reply:
done